### ISSUE TITLE
Camera switch does not work

### ISSUE THREAD

**Issue by JuanmaAranda** [EldertGrootenboer/Managing-StreamYard-using-the-Elgato-Stream-Deck#2]
Oh, that's great! I have downloaded and imported all the scripts and all 7 scene buttons work perfectly.
However, I have not been able to get the camera button to work. When I press it in StreamDeck, it changes color, but does not turn off the camera in StreamYard.
It's strange, because it seems that the keyboard shortcut Ctrl+Alt+May+V is also the one that appears in the AutoKey and Tampermonkey HotKeys....
Does anyone else get this error?
Thanks for everything!

**Comment by paperbone** [comment]
Yes, I'm getting the same results.  I'm poking around in the script and the page source code to try and see if there might be an easy solution...but I don't really know programing.  I would also love to be able to turn on/off backgrounds in streamyard...so I'm trying to modify the code to add that...no luck yet, though.

**Comment by paperbone** [comment]
Ok...I got the camera working for me.  I merely had to change the keystroke from a lowercase v... to a lower case l (I realize this looks like a one...but it's the letter before m)

Don't know why this worked...but it did.  
Just remember to change it in the **tampermonkey script** ... in the **ahk script** (and save and rerun this) ... and then be sure to set it **in your streamdeck** as well.

Anyway...seems to be working fine for me now.

Still trying to figure out how to change a background with a hotkey...no luck on that yet.

**Comment by JuanmaAranda** [comment]
Thanks for the help paperbone!
I tried to do the solution you propose, but in my case it didn't work either. Even if I change the V to l the camera switch doesn't work either.
It's very strange... we have to keep investigating my friend!

**Comment by likamrat** [comment]
All you need to do is to have "Turn" with a capital "T" 

`('[aria-label="Turn on camera"]');`
`('[aria-label="Turn off camera"]');`

**Comment by EldertGrootenboer** [comment]
This has now been updated in the sources as well.

**Comment by JuanmaAranda** [comment]
Does it still work for you? It worked fine for me for a few days, until a Google Chrome update arrived. Since then it stopped working. Today I tried to install it again and it still doesn't work. None of the StreamDeck keys respond in StreamYard.
